ui/src/submissions/conferences/components/ExistingConferencesDrawer.scss

```diff
@@ -1,5 +1,5 @@
 .__ExistingConferencesDrawer__ {
-  z-index: $zindex-header + 1;
+  z-index: $zindex-header + 1 !important;
```

Comment:
do we really need the `!important`?

Reply:
Yes, it's necessary, otherwise it's overwritten by https://github.com/inspirehep/inspirehep/blob/335b060eaf1814c4c17708bbad5730aa0ed16fe3/ui/src/App.scss#L182